test/contracts-network/colony-task.js

```diff
@@ -1293,7 +1294,16 @@ contract("ColonyTask", (accounts) => {
     it("should fail if it's not sufficiently funded to support all its payouts", async () => {
       await fundColonyWithTokens(colony, token, INITIAL_FUNDING);
       const taskId = await makeTask({ colonyNetwork, colony, token });
-      await colony.setAllTaskPayouts(taskId, token.address, MANAGER_PAYOUT, EVALUATOR_PAYOUT, WORKER_PAYOUT, { from: MANAGER });
+
+      await setPayouts({
```

Comment:
If setAllTaskPayouts is back, we can drop these changes too.

Reply:
Alrightttttt, although we could just leave this in so we don't have to redo it later.